ccd-definition/CaseEventToComplexTypes/draft-SDO/noticeOfProceedings.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "NoticeOfProceedings",
+    "CaseEventID": "draftSDO",
+    "CaseFieldID": "noticeOfProceedings",
+    "ListElementCode": "proceedingTypes",
+    "EventElementLabel": "Which notice of hearings need to be sent with this order?",
```

Comment:
should this be 'notice of proceedings'?

Reply:
Attached screen says ' notice of hearings' but this is a typo in the attached screens.